lib/views/Account/V4Deposits/DelegateView.tsx

```diff
@@ -0,0 +1,280 @@
+import {
+  BlockExplorerLink,
+  ModalTitle,
+  SquareButton,
+  SquareButtonTheme,
+  ThemedClipSpinner
+} from '@pooltogether/react-components'
+import FeatherIcon from 'feather-icons-react'
+import classNames from 'classnames'
+import { ethers } from 'ethers'
+import { useState } from 'react'
+import { FieldValues, useForm, UseFormRegister } from 'react-hook-form'
+import { useTranslation } from 'react-i18next'
+import { PrizePool } from '@pooltogether/v4-js-client'
+import { useTransaction } from '@pooltogether/hooks'
+
+import { useUsersAddress } from 'lib/hooks/useUsersAddress'
+import { useUsersTicketDelegate } from 'lib/hooks/v4/PrizePool/useUsersTicketDelegate'
+import { DepositItemsProps } from '.'
+import { useUser } from 'lib/hooks/v4/User/useUser'
+import { useSendTransaction } from 'lib/hooks/useSendTransaction'
+import { InfoList } from 'lib/components/InfoList'
+import { TxHashRow } from 'lib/components/TxHashRow'
+import { useIsWalletOnNetwork } from 'lib/hooks/useIsWalletOnNetwork'
+import { TxButtonNetworkGated } from 'lib/components/Input/TxButtonNetworkGated'
+
+const DELEGATE_ADDRESS_KEY = 'delegate'
+
+interface DelegateViewProps extends DepositItemsProps {}
+
+enum DelegateViews {
+  read = 'read',
+  write = 'write'
+}
+
+export const DelegateView = (props: DelegateViewProps) => {
+  const { prizePool } = props
+  const { t } = useTranslation()
+  const usersAddress = useUsersAddress()
+  const { data: delegate, isFetched, refetch } = useUsersTicketDelegate(usersAddress, prizePool)
+  const [view, setView] = useState<DelegateViews>(DelegateViews.read)
+
+  return (
+    <>
+      <ModalTitle
+        className='mb-4'
+        chainId={prizePool.chainId}
+        title={t('delegateDeposits', 'Delegate deposits')}
```

Comment:
Perhaps rename this to "Delegate deposit", since we can only delegate the entire deposit and not multiple deposits.